pulsar-client-api/src/main/java/org/apache/pulsar/client/api/PulsarClientException.java

```diff
@@ -910,6 +910,28 @@ public TransactionConflictException(String msg) {
         }
     }
 
+    public static class TransactionCanNotCommitException extends PulsarClientException {
```

Comment:
Since both commit or abort might get this exception if users did not handle send message futures or ack message futures properly, I think we should change it to `TransactionOngoingRequestsNotCompleteException`?